.licensed.yml

```diff
@@ -0,0 +1,15 @@
+sources:
```

Comment:
How is this file generated and used? Do we have a list of all acceptable licenses?

For example, I saw a `- 0bsd` license in one of the sibling PRs and it's not here.  If I am contributing, I maybe a bit confused for my next step.  I wouldn't know if I should add the license here manually or it means that the lib is not acceptable in this project?